subprojects/core/src/main/java/org/gradle/api/artifacts/ResolvableDependencies.java

```diff
@@ -61,22 +57,6 @@
     FileCollection getFiles();
 
     /**
```

Comment:
We should remove these methods asap. But we might decide not to do it with this PR.